examples/provider/bid/conversant/conversant-ex1.html

```diff
@@ -0,0 +1,379 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Conversant - PubFood Example page</title>
+  </head>
+  <body>
+    <script src="../../../../build/pubfood.js"></script>
+    <script  type="text/javascript">
+
+      //add the following lines to 'Code Snippet' box of DFP third party creative within a <script> tag
+      //gpt would then call renderAd below to render the ad
+
+     /*
+      var w = window;
+      for (i = 0; i < 10; i++) {
+        w = w.parent;
+        try {
+          w.renderAd(document, '%%PATTERN:cn728x90adId%%');
+          break;
+        } catch(e) {continue;}
+      }
+     */
+
+      //rudimentary html validation, use stricter validation if needed
+      var validateHTML = function (snippet) {
+        return (snippet && snippet.startsWith('<') && snippet.endsWith('>'));
+      };
+      //use similar logic to render ad. dfp would call with document and id
+      window.renderAd = function (doc, id) {
+        if (doc && id) {
+          try {
+            //lookup ad by ad Id
+            var adObject = savedBids.find(function(bid) {return bid.id === id;});
```

Comment:
[Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility)  still potentially not widely supported.

Rather than a polyfill, some simple JavaScript is preferred.

When the project makes more use of transpilation, later ECMA standards will be more acceptable.